internal/uvm/network.go

```diff
@@ -314,3 +328,16 @@ func (uvm *UtilityVM) removeNIC(ctx context.Context, id guid.GUID, endpoint *hns
 	}
 	return nil
 }
+
+// Removes all Network namespaces and corresponding enpoints associated with this UVM
```

Comment:
Don't we have a function similar? RemoveNetNS removes all NICs and issues a guest request to remove the namespace. Might be missing something here so let me know.

Reply:
We don't want to remove the namespace in this case, we just want to remove the NICs.

Reply:
Can we update the comment then?